.github/workflows/ios-sdk-release.yml

```diff
@@ -42,17 +42,16 @@ jobs:
       - name: Install Cocoapods
         run: gem install cocoapods
 
-      - name: set version number and push podspecs to git
-        run: bundle exec fastlane ios bump_release_version version:$VERSION
-
```

Comment:
removing this as a separate step to be done on a branch and merge prior to GA workflow run